feat(genapi): init docs (#3636)
scaleway/docs-content: ai-data/generative-apis/api-cli/using-models-api.mdx

---
meta:
  title: Using Models API
  description: This page explains how to use the Models API
content:
  h1: Using Models API
  paragraph: This page explains how to use the Models API
tags: generative-apis ai-data embeddings-api
dates:
  validation: 2024-09-02
  posted: 2024-09-02
---

Scaleway Generative APIs are designed as drop-in replacement for the OpenAI APIs.

> Using the Models API, it is easy to list the various AI models available at Scaleway.

Reviewer: ```suggestion
The Models API allows you to easily list the various AI models available at Scaleway.
```

## List models

> Lists the available models, and provides basic information about each one.

Reviewer: ```suggestion
Run the following command to list the available models with basic information about each one:
```

Reviewer 2: Same comment
